ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -68,11 +68,10 @@ message CreatedEvent {
   // Optional
   Record create_arguments = 4;
 
-  // Opaque representation of contract payload intended for forwarding
-  // to an API server as a contract disclosed as part of a command
-  // submission.
+  // Deprecated. Use the data retrieved in ``create_arguments`` field
+  // for submitting as a disclosed contract with future commands.
   // Optional
-  google.protobuf.Any create_arguments_blob = 12;
+  google.protobuf.Any create_arguments_blob = 12 [deprecated = true];
```

Comment:
Is this the target state for 2.8 or is this an intermediate step to simplify the rollout?

When do you plan to remove the support completely?